api/serverless.ts

```diff
@@ -317,6 +319,30 @@ serverlessConfiguration.functions = {
         }
       : undefined
   ),
+
+  migrateBusinessesOnUserChange: migrateBusinessesOnUserChange(
+    env.CI
+      ? {
+          securityGroupIds: ["${self:custom.config.infrastructure.SECURITY_GROUP}"],
+          subnetIds: [
+            "${self:custom.config.infrastructure.SUBNET_01}",
+            "${self:custom.config.infrastructure.SUBNET_02}",
+          ],
+        }
+      : undefined
+  ),
+
+  oneTimeBusinessMigration: oneTimeBusinessMigration(
+    env.CI
+      ? {
+          securityGroupIds: ["${self:custom.config.infrastructure.SECURITY_GROUP}"],
+          subnetIds: [
+            "${self:custom.config.infrastructure.SUBNET_01}",
+            "${self:custom.config.infrastructure.SUBNET_02}",
+          ],
+        }
+      : undefined
+  ),
```

Comment:
[dust] We're still setting up the one-time migration lambda here. Is this still needed now that we've moved our approach to a migrations-based one?

Reply:
Yes, the one-time migration is still necessary. The current migrations focus solely on modifying the UserData shape. This particular migration, however, involves copying data to the new table, which is a distinct operation not covered by the existing approach. Therefore, retaining the one-time migration is essential to ensure this data transfer is properly handled.